.github/workflows/make_wheel_macOS_arm64.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Additional change for CI pipeline is needed, please refer to 
make whl: https://github.com/tensorflow/addons/blob/master/.github/workflows/release.yml#L45
upload whl: https://github.com/tensorflow/addons/blob/master/.github/workflows/release.yml#L89